e2e-tests/portal/cypress/integration/journeys/maker/reports/audit-supply-contracts/report-should-only-show-data-from-users-bank.spec.js

```diff
@@ -29,27 +22,20 @@ context('Audit - Report', () => {
   });
 
   before(() => {
-    cy.deleteDeals(BARCLAYS_LOGIN);
-    cy.deleteDeals(HSBC_MAKER_1);
-    cy.deleteDeals(HSBC_MAKER_2);
-
-    cy.insertManyDeals([aDealWithOneBond, aDealWithTenLoans], BARCLAYS_LOGIN)
-      .then((insertedDeals) => barclaysDeals = insertedDeals);
+    cy.deleteDeals(BANK1_MAKER);
+    cy.deleteDeals(BANK2_MAKER);
 
-    cy.insertManyDeals([aDealWithTenLoansAndTenBonds, aDealWithOneLoan], HSBC_MAKER_1)
-      .then((insertedDeals) => hsbcDeals.push(insertedDeals));
+    cy.insertManyDeals([aDealWithOneBond, aDealWithTenLoans], BANK1_MAKER);
 
-    cy.insertManyDeals([aDealWithOneLoanAndOneBond, aDealWithTenLoans], HSBC_MAKER_2)
-      .then((insertedDeals) => hsbcDeals.push(insertedDeals));
+    cy.insertManyDeals([aDealWithTenLoansAndTenBonds, aDealWithOneLoan], BANK2_MAKER);
```

Comment:
We've lost the test where a maker of one bank can see deals made by a maker of same bank

Reply:
Good spot, i've updated the test here: https://github.com/foundry4/dtfs2/pull/197/commits/43da29f634f1937e6e206c83cecdffa75cc68f9d